.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+<!-- Please describe your changes -->
+
+## How Has This Been Tested?
+<!-- Please describe in detail how you tested your changes. -->
+<!-- Include details of your testing environment, tests ran to see how -->
+<!-- your change affects other areas of the code, etc. -->
+
+## Screenshots (jpeg or gifs if applicable):
+
+## Types of changes
+<!-- What types of changes does your code introduce?  -->
+<!-- Bug fix (non-breaking change which fixes an issue) -->
+<!-- New feature (non-breaking change which adds functionality) -->
+<!-- Breaking change (fix or feature that would cause existing functionality to not work as expected) -->
+
+## Checklist:
+- [ ] My code is tested.
+- [ ] My code follows the WordPress code style.
+- [ ] My code follows has proper inline documentation.
```

Comment:
Same questions here. Do we really want links within the PR template? I haven't seen any other templates include links like this before. If it's not in an HTML comment you're going to see a bunch of people not remove it and clutter up the actual content.